frappe/integrations/doctype/google_drive/google_drive.py

```diff
@@ -197,6 +197,7 @@ def upload_system_backup_to_google_drive():
 			file_urls.append(backup.backup_path_files)
 			file_urls.append(backup.backup_path_private_files)
 	else:
+		take_files_backup()
```

Comment:
```suggestion
		backup_files()
```

Reply:
This isnt zip_flies, it is a seperate function.
Also backup_files() is not in v-12